### ISSUE TITLE
AbstractMethodError in onPlayerStateChanged

### ISSUE THREAD

**Issue by deepakpsad** [rever-ai/Goonj#4]
```
Process: com.capitalx.blissfully, PID: 31964
    java.lang.AbstractMethodError: abstract method "void com.google.android.exoplayer2.analytics.AnalyticsListener.onPlayerStateChanged(com.google.android.exoplayer2.analytics.AnalyticsListener$EventTime, boolean, int)"
        at com.google.android.exoplayer2.analytics.AnalyticsCollector.onPlayerStateChanged(AnalyticsCollector.java:470)
        at com.google.android.exoplayer2.ExoPlayerImpl$PlaybackInfoUpdate.lambda$run$4$ExoPlayerImpl$PlaybackInfoUpdate(ExoPlayerImpl.java:812)
        at com.google.android.exoplayer2.-$$Lambda$ExoPlayerImpl$PlaybackInfoUpdate$sJrY7lA_vUJy5MdfV-ndTSxVTXI.invokeListener(Unknown Source:2)
        at com.google.android.exoplayer2.BasePlayer$ListenerHolder.invoke(BasePlayer.java:165)
        at com.google.android.exoplayer2.ExoPlayerImpl.invokeAll(ExoPlayerImpl.java:823)
        at com.google.android.exoplayer2.ExoPlayerImpl.access$000(ExoPlayerImpl.java:43)
        at com.google.android.exoplayer2.ExoPlayerImpl$PlaybackInfoUpdate.run(ExoPlayerImpl.java:810)
        at com.google.android.exoplayer2.ExoPlayerImpl.notifyListeners(ExoPlayerImpl.java:726)
        at com.google.android.exoplayer2.ExoPlayerImpl.updatePlaybackInfo(ExoPlayerImpl.java:701)
        at com.google.android.exoplayer2.ExoPlayerImpl.prepare(ExoPlayerImpl.java:232)
        at com.google.android.exoplayer2.SimpleExoPlayer.prepare(SimpleExoPlayer.java:913)
        at com.google.android.exoplayer2.SimpleExoPlayer.prepare(SimpleExoPlayer.java:898)
        at ai.rever.goonj.player.LocalAudioPlayer.enqueue(LocalAudioPlayer.kt:236)
        at ai.rever.goonj.manager.GoonjPlayerManager.addTrack(GoonjPlayerManager.kt:79)
        at ai.rever.goonj.manager.GoonjPlayerManager.addTrack$default(GoonjPlayerManager.kt:75)
        at ai.rever.goonj.Goonj$addTrack$1.invoke(Goonj.kt:149)
        at ai.rever.goonj.Goonj$addTrack$1.invoke(Goonj.kt:32)
        at ai.rever.goonj.Goonj.run(Goonj.kt:62)
        at ai.rever.goonj.Goonj.addTrack(Goonj.kt:146)
        at ai.rever.goonj.interfaces.GoonjPlayer$DefaultImpls.addTrack(GoonjPlayer.kt:26)
        at ly.blissful.bliss.ui.commons.player.audio.v2.GoonjPlayerActivity.addTrack(GoonjPlayerActivity.kt:28)
        at ai.rever.goonj.interfaces.GoonjPlayer$DefaultImpls.addTrack$default(GoonjPlayer.kt:26)
        at ly.blissful.bliss.ui.commons.player.audio.v2.GoonjPlayerActivity.playNewTrack(GoonjPlayerActivity.kt:210)
        at ly.blissful.bliss.ui.commons.player.audio.v2.GoonjPlayerActivity$requestStart$2.accept(GoonjPlayerActivity.kt:192)
        at ly.blissful.bliss.ui.commons.player.audio.v2.GoonjPlayerActivity$requestStart$2.accept(GoonjPlayerActivity.kt:28)
        at io.reactivex.internal.operators.maybe.MaybeCallbackObserver.onSuccess(MaybeCallbackObserver.java:71)
        at io.reactivex.internal.operators.maybe.MaybeMap$MapMaybeObserver.onSuccess(MaybeMap.java:89)
        at io.reactivex.internal.operators.maybe.MaybeFilter$FilterMaybeObserver.onSuccess(MaybeFilter.java:89)
        at io.reactivex.internal.operators.maybe.MaybeCreate$Emitter.onSuccess(MaybeCreate.java:73)
        at durdinapps.rxfirebase2.RxFirestore$14$2.onSuccess(RxFirestore.java:499)
        at durdinapps.rxfirebase2.RxFirestore$14$2.onSuccess(RxFirestore.java:495)
        at com.google.android.gms.tasks.zzn.run(Unknown Source:4)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6854)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:860)
```

**Comment by kshivang** [comment]
This error may or may not occur, depend on build(may be related to jitpack). One workaround could be importing Goonj module in project directly

**Comment by kshivang** [comment]
fixed in v0.5.8
